src/main/java/cinema/service/MovieSessionService.java

```diff
@@ -11,5 +12,7 @@
 
     void delete(Long id);
 
+    Optional<MovieSession> getById(Long id);
```

Comment:
```suggestion
    MovieSession getById(Long id);
```
return Optional from the dao layer, and entity from the service layer